File: modul 6/soal 4/PRAK504-2210817220034-Wanda Karliyanti.py
def reverse(a):
    b=0
    while(a!=0):
        b=b*10
        b=b+a%10
        a=a/10
    return b
A, B=map(int,input().split())
A=reverse(A)
B=reverse(B)
C=A+B
print(reverse(C))

File: modul 6/soal 4/PRAK504-2210817220034-Wanda Karliyanti.c
#include <stdio.h>
int reverse(int a){
    int b=0;
    while(a!=0){
        b=b*10;
        b=b+a%10;
        a=a/10;
    }
    return b;
}
    int main() {
        int A, B;
        scanf("%d %d",&A,&B);
        A=reverse(A);
        B=reverse(B);
        int C = A+B;
        printf("%d",reverse(C));
}
